0xSpaceShard/starknet-bridge-relayer
# Feat/cron module

### Description

Added the cron service that allows the relayer automatically as a worker.

### Demo
![ex](https://user-images.githubusercontent.com/127413534/228555148-f787363a-91fe-4491-96e0-00cdecf1bd91.PNG)

### Testing notes
I removed the volume for the mongo database, the idea is to hae a state less env for testing. Also, reduce the number of blocks to index so you don't have to wait lots of time for indexing. The good number should be between 150 and 200 block.
1. Duplicate `example.env` and rename `.env` make sure to add required values like alchemy RPC url.
2. Run `yarn test`
3. Run `make dev-up`
4. Run `make dev-down`

### Checklist:

- [X]  Created / updated tests (e2e / unit).
